static/app/components/dropdownMenuItemV2.tsx

```diff
@@ -105,137 +111,160 @@ type Props = {
   renderAs?: React.ElementType;
 };
 
+/**
+ * Wraps withRouter and forwardRef around Component
+ */
+const withRouterForwardRef = Component => {
```

Comment:
can we add a type here?

Reply:
actually I found a way to pass `submenuTriggerRef` down as a prop, without using `forwardRef`, so we can just remove this function.